atom/browser/api/atom_api_web_contents.cc

```diff
@@ -991,6 +991,9 @@ void WebContents::TabDetachedAt(content::WebContents* contents, int index) {
 
   if (owner_window() && owner_window()->browser())
     owner_window()->browser()->tab_strip_model()->RemoveObserver(this);
+
+  auto api_web_contents = CreateFrom(isolate(), contents);
```

Comment:
you don't need this here because `contents != web_contents()` ensures that each atom_api_web_contents instance only handles the event for itself